### PR TITLE
cruft

### PR DESCRIPTION
**Pull request by bandophahita**
Using [cruft](https://cruft.github.io/cruft/) to help synchronize template portions of each project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ ScreenPy Selenium
...
+[![Supported Versions](https://img.shields.io/pypi/pyversions/screenpy_selenium.svg)](https://pypi.org/project/screenpy_selenium)
 [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)
 [![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
 
 ```
 TITLE CARD:
-                               "ScreenPy Selenium"
+                              "ScreenPy Selenium"
```
Comment (perrygoy) [inline README.md:15]: Such a minor thing, but this trailing empty space is gonna be removed by my text editor on my next commit i promise you.

File: pyproject.toml [hunk 2]
```diff
@@ -175,9 +176,11 @@ importlib_metadata = {version = "*", python = "3.8.*"}
 # convenience packages for development
 black = {version = "*", optional = true}
 coverage = {version = "*", optional = true}
+cruft = {version = "*", optional = true}
 mypy = {version = "*", optional = true}
 pre-commit = {version = "*", optional = true}
 pytest = {version = "*", optional = true}
+pytest-mock = {version = "*", optional = true}
```
Comment (perrygoy) [inline pyproject.toml:183]: Do we use `pytest-mock` in this repo?

**Review by perrygoy — COMMENTED** [review]
Couple minor things just to check

File: pyproject.toml (hunk 2 above)
Comment (bandophahita) [reply pyproject.toml:183]: No.  Half the repos use it, half don't and I felt weird about overriding so many.

**Review by perrygoy — APPROVED** [review]
Concern addressed! I don't think i mind having an extra `pytest-mock` import.

**Pull request merged**